.github/workflows/cibuildwheel.yml

```diff
@@ -1,3 +1,15 @@
+# This CI run builds wheels using the cibuildwheel infrastructure.
+# This means it:
+# - Uses as old dependencies/infrastructure as possible:
```

Comment:
> Are these for old arch/OSes ?

This comment was meant for this part. Yes it's for as far backward in OS compat as we can get